### DISCUSSION TITLE
MVVM best practice

### DISCUSSION THREAD

**Discussion by HppZ** [microsoft/WindowsAppSDK#1172] [Q&A] [answered]
1, Can I put command in Model, when?
2, If I have a Button in View, click the Button to scroll view to top, should I put the logic in cs file or ViewModel?
3, How ViewModel control View? for example, scroll view to top.

**Comment by btueffers** [comment, marked as answer]
@HppZ this sounds like a good question for the WinUI team. They would be able to provide more insight if you opened an Issue in their repo 😊 https://github.com/microsoft/microsoft-ui-xaml
